portal_convert.go

```diff
@@ -357,8 +357,7 @@ func (puppet *Puppet) addMemberMeta(part *ConvertedMessage, msg *discordgo.Messa
 	}
 	if msg.Member.Nick != "" || !avatarURL.IsEmpty() {
 		perMessageProfile := map[string]any{
-			"is_multiple_users": false,
-
+			"id": msg.Author.ID,
```

Comment:
Hmm, this could just be an empty string or a hardcoded one, since there are no other users, it's always the same user